File: Java/ws-java/soma_vetor/src/soma_vetor.java
import java.util.Locale;
import java.util.Scanner;

public class soma_vetor {

	public static void main(String[] args) {
	Locale.setDefault(Locale.US);
	Scanner sc = new Scanner(System.in);
	
	int n;
	double soma, media;
	
	System.out.print("Quantos numeros voce vai digitar? ");
	n = sc.nextInt();
	
	double[] vet = new double[n];
	
	soma = 0;
	for (int i = 0; i < n; i++) {
		System.out.print("Digite um numero: ");
		vet[i] = sc.nextDouble();
		soma = soma + vet[i];
	}
	
	media = (double)soma / n;
	
	System.out.println();
	System.out.print("VALORES = ");
	
	for (int i = 0; i < n; i++) {
		System.out.print(String.format("%.1f", vet[i]) + "  ");
	}
	
	System.out.println();
	System.out.println("SOMA = " + String.format("%.2f", soma));
	System.out.println("MEDIA = " + String.format("%.2f", media));
	
	sc.close();	

	}

}

File: Python/somavetor.py
n = int(input("Quantos numeros você vai digitar: "))

vet = [0 for x in range(n)]

for i in range(0, n):
    print("Digite um numero: ", end="")
    vet[i] = float(input())

soma = 0
print(f"VALORES = ", end="")
for i in range(0, n):
    soma = soma + vet[i]
    print(f"  {vet[i]:.1f}", end="")

print()
media = soma / n
print(f"SOMA = {soma:.2f}")
print(f"MEDIA = {media:.2f}")

